source/help/apps/desktop-changelog.rst

```diff
@@ -6,7 +6,16 @@ Release v4.4.0
 
 **Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/tag/v4.4.0>`_
 
+- **v4.4.1, release day TBD**
+ - Fixing an issue where the Desktop client opens to a blank white Window when using GPO-set teams. `MM-23082 <https://mattermost.atlassian.net/browse/MM-23082>`_
+ - Fixing an issue where Google oAuth with Gmail addresses no longer works on the Desktop app. `MM-23057 <https://mattermost.atlassian.net/browse/MM-23057>`_
+ - Fixing an issue where the app sometimes doesn't restore to the right position but "jumps" to a different place in the taskbar when minimizing the app and then maximizing it. `MM-23195 <https://mattermost.atlassian.net/browse/MM-23195>`_
+ - Fixing an issue where users are not able to paste text into the login screen. `MM-23784 <https://mattermost.atlassian.net/browse/MM-23784>`_
+ - Fixing an issue where back/forward navigation on the OAuth window causes the app to crash. `MM-23153 <https://mattermost.atlassian.net/browse/MM-23153>`_
+ - Fixing an issue where a Desktop notification is not shown when session expires on Windows 10. `MM-23806 <https://mattermost.atlassian.net/browse/MM-23806>`_
```

Comment:
I haven't yet been able to repro this, but by preventing the os from displaying notifications (and then I don't get any but that's the expected behaviour)

Reply:
I removed it for now and it can be added back later if a fix for is done